README.rst

```diff
@@ -690,6 +723,27 @@ Delete a notification
 
     DELETE /api/v1/notification/{pk}/
 
+List user's notification setting
+################################
+
+.. code-block:: text
+
+    GET /api/v1/notifications/setting/
```

Comment:
`notifications` > `notification`.

I also think it would be more explicit to name it user-setting